README.md

```diff
@@ -1,2 +1,57 @@
-# ods-e2e-tests
-End-to-end functional tests, based on Spock/Geb and gradle to assert the correct functionality of an OpenDevStack installation
+# Automated test cases for ODS
+
+## Description
+Suite of test cases for OpenDevStack and the Release Manager.
```

Comment:
@jorge-romero The Release Manager is part of OpenDevStack, no need to add it as an extra item.

Reply:
I will fix the comments